dali/operators/reader/loader/video_loader.cc

```diff
@@ -607,18 +605,6 @@ void VideoLoader::read_file() {
         continue;
     }
 
-    if (frame >= req.frame && frame < req.frame + req.count) {
-      if (frames_read[frame - req.frame]) {
-        ERROR_LOG << "Frame " << frame << " appeared twice\n";
-      } else {
-        frames_read[frame - req.frame] = true;
-        frames_left--;
```

Comment:
What about this piece?

Reply:
Same. As the reader can rewind this part seems to be no longer relevant.

Reply:
I trust you that it's tested and works.

Reply:
This logic is only for logging. I don't see it used for the actual flow.